code/game/dna/genes/vg_powers.dm

```diff
@@ -39,69 +39,99 @@
 	var/mob/living/carbon/human/M=usr
 	var/obj/item/organ/external/head/head_organ = M.get_organ("head")
 
-	var/new_facial = input("Please select facial hair color.", "Character Generation",rgb(head_organ.r_facial, head_organ.g_facial, head_organ.b_facial)) as null|color
-	if(new_facial)
-		head_organ.r_facial = hex2num(copytext(new_facial, 2, 4))
-		head_organ.g_facial = hex2num(copytext(new_facial, 4, 6))
-		head_organ.b_facial = hex2num(copytext(new_facial, 6, 8))
-
-	var/new_hair = input("Please select hair color.", "Character Generation",rgb(head_organ.r_hair, head_organ.g_hair, head_organ.b_hair)) as null|color
-	if(new_facial)
-		head_organ.r_hair = hex2num(copytext(new_hair, 2, 4))
-		head_organ.g_hair = hex2num(copytext(new_hair, 4, 6))
-		head_organ.b_hair = hex2num(copytext(new_hair, 6, 8))
+	var/new_gender = alert(usr, "Please select gender.", "Character Generation", "Male", "Female")
```

Comment:
Moved Gender up top so if someone wants to turn into a male and have a beard, they can do so in one cast of morph.